stage.py

```diff
@@ -39,13 +50,65 @@ def verbose_str(self):
         max_index = i
     return "%s\n    Longest Task: %s" % (self, self.tasks[i])
 
-  def load_balancing_badness(self):
+  def get_executor_id_to_resource_metrics(self):
+    """
+    Returns a mapping from executor id to a description of how each of its resources was used while
```

Comment:
Can you fix these to have a first sentence that fits on a single line, and then longer description below?